BUILD

```diff
@@ -217,21 +216,28 @@ docker_container_image(
     visibility = ["//test:__subpackages__"],
 )
 
-docker_container_bundle(
```

Comment:
container_bundle does not support file-based tag specifications, therefore it's preferable to use `container_push` twice in order to read the tags from the file directly, instead of using the workspace_status workaround.